src/instructlab/model/backends/llama_cpp.py

```diff
@@ -130,6 +123,10 @@ def server(
     queue=None,
 ):
     """Start OpenAI-compatible server"""
+    # create a temporary, throw-away logger
+    host_port = f"{host}:{port}"
+    server_logger = logging.getLogger(host_port)
+    server_logger.setLevel(logging.FATAL)
```

Comment:
The server_logger is set to FATAL level and then the server_logger is used to set the verbose value to False since the server_logger is configured to FATAL. So verbose is always False.